docs/getting-started/tutorial.md

```diff
@@ -522,7 +526,7 @@ export default function Edit( { attributes, setAttributes } ) {
 			<InspectorControls>
 				<PanelBody title={ __( 'Settings', 'copyright-date-block' ) }>
 					<ToggleControl
-						checked={ showStartingYear }
+						checked={ !! showStartingYear }
```

Comment:
It may also be necessary to explain why we explicitly convert it to a boolean type.

Reply:
I think these tutorials expected some JS and React knowledge, so we can probably skip this.